### ISSUE TITLE
(X-mod) construct armory breaking casting in fossils mod material

### ISSUE THREAD

**Issue by mailleseraph** [TheIllusiveC4/ConstructsArmory#149]
#### Versions (Be specific, do not write "latest"):
* Construct's Armory: 1.2.3
* Tinkers' Construct: 2.12.0.115
* Mantle: 1.3.3.39
* Forge: 14.23.5.2772
*Fossils and archeology revival: 8.0.0
*LLibrary: 1.7.17
*JEI: 4.13.1.220

#### Expected Behavior: without constructs armory, tar parts can be cast from the mod fossil and archeology 

Issue on fossil and archeology: https://github.com/FossilsArcheologyRevival/FossilsArcheologyRevival/issues/1265

#### Steps to Reproduce:
1. install mods above, except for constructs armory
2. tar parts are valid
3. install constructs armory, no more tar parts

Crash Log: not a crash issue

**Comment by TheIllusiveC4** [comment]
Hm, what an odd problem. I have no idea what's going on here. I don't see any errors during initialization, and nothing about the code is sticking out to me. I'll have to look into this deeper and get back to you.

**Comment by TheIllusiveC4** [comment]
I've found a solution for the problem that I've posted in the Fossils issue thread. I'm still not sure why Construct's Armory is the only mod triggering this result, but I'm not convinced yet that I'm able to do anything about this on my end. Construct's Armory only adds to materials that are already present in base Tinkers' Construct, so there's not much room for error in regards to affecting third party materials as far as I can see. I'm closing this issue for now, but feel free to let me know if new information arises about the conflict.
